examples/step-6/doc/results.dox

```diff
@@ -484,7 +484,43 @@ step-49, step-53, and step-54, among other tutorial programs that cover
 the issue. Information on curved domains can also be found in the
 documentation module on @ref manifold "Manifold descriptions".
 
-
+Why does it make sense to choose a mesh that tracks the internal
+interface? There are a number of reasons, but the most essential one
+comes down to what we actually integrate in our bilinear
+form. Conceptually, we want to integrate the term $A_{ij}^K=\int_K
+a(\mathbf x) \varphi_i(\mathbf x) \varphi_j(\mathbf x) ; dx$ as the
+contribution of cell $K$ to the matrix entry $A_{ij}$. We can not
+compute it exactly and have to resort to quadrature. We know that
+quadrature is accurate if the integrand is smooth. That is because
+quadrature in essence computes a polynomial approximation to the
+integrand that coincides with the integrand in the quadrature points,
+and then computes the volume under this polynomial as an approximation
+to the volume under the original integrand. This polynomial
+interpolant is accurate if the integrand is smooth on a cell, but it
+is usually rather inaccurate if the integrand is discontinuous on a
+cell.
+
+Consequently, it is worthwhile to align cells in such a way that the
+interfaces across which the coefficient is discontinuous are aligned
+with cell interfaces.  This way, the coefficient is constant on each
+cell, following which the integrand will be smooth, and its polynomial
+approximation and the quadrature approximation of the integral will
+both be accurate.  Note that such an alignment is common in many
```

Comment:
Fair comment. Its looks like we're not consistent in this regard throughout the documentation.